components/raftstore/src/store/worker/region.rs

```diff
@@ -51,24 +45,10 @@ use crate::{
 };
 
 const CLEANUP_MAX_REGION_COUNT: usize = 64;
-const SNAP_GENERATOR_MAX_POOL_SIZE: usize = 16;
-
-const TIFLASH: &str = "tiflash";
-const ENGINE: &str = "engine";
 
 /// Region related task
 #[derive(Debug)]
-pub enum Task<S> {
```

Comment:
Good catch! IMO, the engine trait generics in TiKV are somewhat over-engineered.

Reply:
I didn't catch this, the compiler did 😅 We can remove `<S>` now because `kv_snap: S,` is moved out.